src/blocks/remote-data-container/hooks/useRemoteData.ts

```diff
@@ -194,6 +199,7 @@ export function useRemoteData( {
 			block_name: blockName,
 			query_key: queryKey,
 			query_inputs: inputs,
+			post_id: postId ?? null,
```

Comment:
Very minor but appreciate keeping things in alpha order when that is the current state
```suggestion
			post_id: postId ?? null,
			query_key: queryKey,
			query_inputs: inputs,
```